tests/tools/oecert/host/host.cpp

```diff
@@ -46,6 +51,7 @@ typedef struct _input_params
 } input_params_t;
 
 static input_params_t _params;
+static char* enclave_filename;
```

Comment:
enclave_filename does not need to be global. If `_get_enclave_filename` returns the allocated string, it can be called, verified, can consumed in `main()`.

Reasons

- Global variables should be used conservatively
- `_parse_args` calls `_get_enclave_filename` but `_get_enclave_filename` is not one of the args
-

Reply:
Make sense. Updated `_get_enclave_filename()`